packages/node_modules/@webex/internal-plugin-mercury/test/integration/spec/mercury.js

```diff
@@ -69,7 +70,7 @@ describe('plugin-mercury', function () {
       });
 
       describe('when web-high-availability is enabled', () => {
-        it('connects to mercury using service catalog url', () => {
+        flaky(it)('connects to mercury using service catalog url', () => {
```

Comment:
This may not work in spawned browsers because the environment is different. I've had to explicitly check like `process.env.SKIP_FLAKY_TESTS ? it.skip : it`